cheats/javascript.cheat

```diff
@@ -21,6 +21,38 @@ npm install <package_name> --save-dev
 # install globally a specified package
 npm install <package_name> -g
 
+# run a script
+npm run <script>
+
+$ script: cat package.json | python -c "import sys, json; print('\n'.join(json.load(sys.stdin)['scripts'].keys()))"
```

Comment:
```
node -p "Object.keys(require('./package.json').scripts).join('\n')"
```

If you're searching for `node` snippets, you'll probably have `node`, not necessarily `python` 😃

Reply:
lol, that makes sense. Let me update it

Reply:
Also, you don't need the `cat` now, the `require('./package.json')` part reads the package.json file for you 🙃

Reply:
Thanks. I forgot that